parent/pom.xml

```diff
@@ -115,8 +115,8 @@
     <slf4j-log4j12.version>1.7.0</slf4j-log4j12.version>
     <reactor-core.version>3.3.0.RELEASE</reactor-core.version>
     <reactor-netty.version>0.9.0.RELEASE</reactor-netty.version>
-    <netty.version>4.1.39.Final</netty.version>
-    <okhttp.version>4.1.0</okhttp.version>
+    <netty.version>4.1.42.Final</netty.version>
```

Comment:
The version of reactor-core we're using is 0.9.0-RELEASE which depends on 4.1.39.Final version of Netty. I'm not sure about any issue with we using a version of netty different than reactor-core depends on.

Reply:
It's a minor version upgrade and their release notes mention that 41, 42 are bug-fix releases with performance improvements. 

There's at least [one bug](https://github.com/Azure/azure-sdk-for-java/issues/5170) that was reported by a customer and fixed in `41.Final` release . So, I would prefer to upgrade unless we find issues with `42.Final` that's not in `39.Final`.

https://netty.io/news/2019/09/12/4-1-41-Final.html
https://netty.io/news/2019/09/25/4-1-42-Final.html